### PR TITLE
Bucket / customizer dqa adjustments

### PR DESCRIPTION
**Pull request by alliuca**
**Description:** Merges the Customizer DQA adjustments bucket branch:
- Updates the customizer's custom headings font size

**Ticket:** https://moderntribe.atlassian.net/browse/TEC-3696

**Screenshots:**

Screen Shot 2020-12-10 at 3 15 42 PM

Screen Shot 2020-12-10 at 3 15 51 PM

### CODE REVIEW

File: src/Tribe/Customizer/Controls/Heading.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ class Heading extends Control {
 	 */
 	public function render_content() {
 		?>
-		<h4 style="font-size: 16px; font-weight: normal; line-height: 1.75; margin-top: 0; margin-bottom: 0px;">
+		<h4 style="font-size: 20px; font-weight: normal; line-height: 1.75; margin-top: 0; margin-bottom: 0px;">
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Controls/Heading.php:47]: * :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 112 characters
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: PHP files must only contain PHP code

<sub>hash: aa502ad6df9c157176edfa3a440e8d8e</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 3 error(s)

**Comment by sc0ttkclark** [comment]
Just noting -- [paul said the PR is not for this release](https://tribe.slack.com/archives/CEUHNUF3R/p1607702433449800?thread_ts=1607701567.449000&cid=CEUHNUF3R) so this can be ignored for B20.12 release purposes

File: src/Tribe/Customizer/Controls/Heading.php (hunk 1 above)
Comment (tr1b0t) [inline src/Tribe/Customizer/Controls/Heading.php:47]: * :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 112 characters
* :no_entry_sign: PHP files must only contain PHP code

<sub>hash: 168f7e5b433ab0f95e18484245ddaff9</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 3 error(s)

**Review by sc0ttkclark — APPROVED** [review]

**Review by paulmskim — COMMENTED** [review]
looks like we'll need to fix tests, but otherwise :+1:

**Pull request merged**